virttest/virsh.py

```diff
@@ -4209,10 +4209,18 @@ def vol_upload(name, dfile, options="", **dargs):
     return command(cmd, **dargs)
 
 
-def blkiotune(name, weight=None, device_weights=None, options=None, **dargs):
+def blkiotune(name, weight=None, device_weights=None, device_read_iops_sec=None,
```

Comment:
I am not sure if we will have other more parameters need to be added in this function. It seems too long. Maybe a param_dict is more simple. But however, it is OK to me so far.